ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -98,6 +98,53 @@ export default class TestDataModal extends React.Component {
     this.processOptions();
   }
 
+  addRelatedApplications = async params => {
```

Comment:
I see that `addRelatedApplications` returns boolean/undefined based on `response.failureStatus`.
This seems a little odd for the caller of this function.

Reply:
`addRelatedApplications` doesn't return anything. It only performs `this.setState()` at the end.

The only `return` statement we have is in the arrow function defined for `response.data.filter()`. But that return doesn't influence the output of `addRelatedApplications`.

Reply:
Sorry, my bad. You're right.